packages/frontend/src/components/Home/RecentlyUpdatedPanel/index.tsx

```diff
@@ -27,31 +27,26 @@ interface Props {
 const RecentlyUpdatedPanel: FC<Props> = ({ projectUuid }) => {
     const history = useHistory();
     const { user, health } = useApp();
-    const { data: dashboards = [] } = useDashboards(projectUuid);
-    const { data: savedCharts = [] } = useSavedCharts(projectUuid);
+    const { data: dashboards = [], isLoading: isDashboardsLoading } =
+        useDashboards(projectUuid);
+    const { data: savedCharts = [], isLoading: isChartsLoading } =
+        useSavedCharts(projectUuid);
 
     const recentItems = useMemo(() => {
         return [
             ...wrapResourceView(dashboards, ResourceViewItemType.DASHBOARD),
             ...wrapResourceView(savedCharts, ResourceViewItemType.CHART),
-        ]
-            .sort((a, b) => {
-                if (isResourceViewSpaceItem(a) || isResourceViewSpaceItem(b)) {
-                    return 0;
-                }
-
-                return (
-                    new Date(b.data.updatedAt).getTime() -
-                    new Date(a.data.updatedAt).getTime()
-                );
-            })
-            .slice(0, 10);
+        ];
     }, [dashboards, savedCharts]);
 
     const handleCreateChart = () => {
         history.push(`/projects/${projectUuid}/tables`);
     };
 
+    if (isDashboardsLoading || isChartsLoading) {
+        return null;
```

Comment:
Lately, I'm trying to use [skeletons](https://blueprintjs.com/docs/#core/components/skeleton) rather than returning null to avoid the UI "jump" when components pop up out of nowhere.

Reply:
we are migrating this component to mantine now so might not be the right time. but I'll consider doing the same with mantine